keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/ScalarDoubleTensor.java

```diff
@@ -18,7 +18,7 @@
 
 import static io.improbable.keanu.tensor.TensorShape.calculateShapeForLengthOneBroadcast;
 
-public class ScalarDoubleTensor implements DoubleTensor {
+public class ScalarDoubleTensor extends DoubleTensor {
```

Comment:
what is the relationship between `ScalarDoubleTensor` and `JVMDoubleTensor`? Do we even need `ScalarDoubleTensor` any more?